servers/dsub/jobs/test/test_jobs_controller_local.py

```diff
@@ -53,6 +53,14 @@ def job_has_status(self, job, status):
             return job.labels['status-detail'] != PROCESS_NOT_FOUND_MESSAGE
         return has_status
 
+    def test_abort_job(self):
+        started = self.start_job('sleep 30')
+        api_job_id = self.get_api_job_id(started)
+        time.sleep(10)
```

Comment:
This is pretty unfortunate actually. Is there a TODO which could improve this?

Reply:
Filed https://github.com/googlegenomics/dsub/issues/101 and added a TODO